benchmark/venv.py

```diff
@@ -193,6 +206,25 @@ def create_virtualenv():
     return venv_python
 
 
+def create_virtualenv_py3(venv, venv_path):
+    class BenchmarkEnv(venv.EnvBuilder):
```

Comment:
Why not using virtualenv module on Python2? Since we use the virtualenv command, the Python 2 virtualenv module must be available not?